### DISCUSSION TITLE
quarkus-cxf 1.7.x (or 1.8.x)?

### DISCUSSION THREAD

**Discussion by famod** [quarkiverse/quarkus-cxf#918] [Ideas]
Hey @quarkiverse/quarkiverse-cxf!

Is it just me or do we have a blind spot w.r.t to supporting Quarkus 2.16?
We released 1.7.3 back in January and while that version is working well with Quarkus 2.16, we don't have a 1.7 branch that we update from time to time and release from.
The consequence is that projects which cannot move to Q3 yet are stuck with a basically "unmaintained" version of quarkus-cxf.

What about adding a 1.7 branch, bring it up to date (with 1.5?) where applicable  and do a timely release?
That would also include an update to cxf 3.5.6 or even 3.5.7.
Alternatively, we could add a 1.8 branch and update cxf right to 3.6.x (I haven't checked the implications of that update though).

WDYT?

**Comment by ppalaga** [comment]
Given that there is Red Hat Build of Quarkus (RHBQ, a product with commercial subscription) based on 2.13 branch whereas there is no RHBQ on 2.16, I'd say that Quarkus 2.13 and Quarkus CXF 1.5 are a safer bet for folks not wanting to migrate to Quarkus 3. But still, I won't stand in way to anybody who would like to maintain QCXF 1.7 for Quarkus 2.16.

**Reply by famod** [reply]
It's not so much about not wanting to migrate to Q3, it's about not being able yet.
